.gitlab-ci.yml

```diff
@@ -2,6 +2,12 @@ stages:
   - test
   - deploy
 
+variables:
```

Comment:
Could a short comment be added here that tells us what the depot path is for? IE for where the julia projects are cached?